### DISCUSSION TITLE
how to use API

### DISCUSSION THREAD

**Discussion by hansen-innovation** [oobabooga/text-generation-webui#1688] [General]
If I want to use this model in a Jupyter notebook utilizing its API, similar to OpenAI's API, what steps should I follow? Thank you very much.

**Comment by LabibaAI** [comment]
Interested

**Comment by punchewwastaken** [comment]
There are a few different examples of API in one-click-installers-main\text-generation-webui, among them stream, chat and stream-chat API examples.
